sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -1,5 +1,4 @@
 """ test the label propagation module """
-
```

Comment:
This should must be reverted.

Reply:
Should I add the blank line or remove following changes in `test_convergene_speed` ?

```
from sklearn.utils._testing import _convert_container
CONSTRUCTOR_TYPES = ("array", "sparse_csr", "sparse_csc")

@pytest.mark.parametrize("constructor_type", CONSTRUCTOR_TYPES)
def test_convergence_speed(constructor_type):
    # This is a non-regression test for #5774
    X = _convert_container([[1.0, 0.0], [0.0, 1.0], [1.0, 2.5]], constructor_type)
    y = np.array([0, 1, -1])
    mdl = label_propagation.LabelSpreading(kernel="rbf", max_iter=5000)
    mdl.fit(X, y)

    # this should converge quickly:
    assert mdl.n_iter_ < 10
    assert_array_equal(mdl.predict(X), [0, 1, 1])
```